### PR TITLE
Fix slider ticks offset

### PR DESCRIPTION
**Pull request by henriquelalves**
I commented out "tickarea" line just as a reference - I'll delete it if necessary.

This should fix issue #14637

### CODE REVIEW

**Comment by akien-mga** [comment]
> I commented out "tickarea" line just as a reference - I'll delete it if necessary.

Yes, please remove it :)

Also, please amend the commit message to be more descriptive. "Fix issue #number" doesn't say anything to someone reading the changelog/git log. You need to be on the GitHub website and click the link to actually understand what was fixed. See https://github.com/godotengine/godot/blob/master/CONTRIBUTING.md#format-your-commit-logs-with-readability-in-mind

**Comment by henriquelalves** [comment]
Damn, I rebased with your commit on it. Sorry, I'll fix it in a min.

**Comment by henriquelalves** [comment]
Done, should be the same now.

File: scene/gui/slider.cpp [hunk 1]
```diff
@@ -184,10 +184,10 @@ void Slider::_notification(int p_what) {
 					focus->draw(ci,Rect2i(Point2i(),Size2i(style->get_minimum_size().width+style->get_center_size().width,size.height)));
 				*/
 				if (ticks > 1) {
-					int tickarea = size.height - tick->get_height();
+					int grabber_offset = (grabber->get_size().height/2 - tick->get_height()/2);
```
Comment (Noshyaar) [inline scene/gui/slider.cpp]: Spaces surrounding operators are mandatory. At `/`. Same in Line 208.

Also it seems like your Git email is not the same as your Github's, so it doesn't link to your account. Is this intended?

Comment (henriquelalves) [reply scene/gui/slider.cpp]: @Noshyaar done! I also linked my email to my github account, didn't notice it wasn't done yet, thanks.

**Comment by Merlin1846** [comment]
Multiple commits are saying this bug is fixed when it isn't. It's still happening and although it has no real affect on the game engine it is a bit annoying.

**Pull request merged**